egeria-graph-repository/berkeley/je.config.csv

```diff
@@ -0,0 +1,4 @@
ner,envcfg:je.env.runEvictor,envcfg:je.env.runINCompressor,envcfg:je.env.runOffHeapEvictor,envcfg:je.env.runVerifier,envcfg:je.env.setupLogger,envcfg:je.env.sharedLatches,envcfg:je.env.startupThreshold,envcfg:je.env.terminateTimeout,envcfg:je.env.ttlClockTolerance,envcfg:je.env.ttlLnPurgeDelay,envcfg:je.env.ttlMaxTxnTime,envcfg:je.env.verifyBtree,envcfg:je.env.verifyDataRecords,envcfg:je.env.verifyLog,envcfg:je.env.verifyObsoleteRecords,envcfg:je.env.verifySchedule,envcfg:je.env.verifySecondaries,envcfg:je.evictor.allowBinDeltas,envcfg:je.evictor.coreThreads,envcfg:je.evictor.criticalPercentage,envcfg:je.evictor.deadlockRetry,envcfg:je.evictor.evictBytes,envcfg:je.evictor.evictionBatchPercentage,envcfg:je.evictor.forcedYield,envcfg:je.evictor.keepAlive,envcfg:je.evictor.lruOnly,envcfg:je.evictor.maxThreads,envcfg:je.evictor.mutateBins,envcfg:je.evictor.nLRULists,envcfg:je.evictor.nodeScanPercentage,envcfg:je.evictor.nodesPerScan,envcfg:je.evictor.useDirtyLRU,envcfg:je.evictor.useMemoryFloor,envcfg:je.evictor.wakeupInterval,envcfg:je.haltOnCommitAfterChecksumException,envcfg:je.lock.deadlockDetect,envcfg:je.lock.deadlockDetectDelay,envcfg:je.lock.nLockTables,envcfg:je.lock.oldLockExceptions,envcfg:je.lock.timeout,envcfg:je.log.bufferSize,envcfg:je.log.checksumRead,envcfg:je.log.chunkedNIO,envcfg:je.log.detectFileDelete,envcfg:je.log.detectFileDeleteInterval,envcfg:je.log.directNIO,envcfg:je.log.faultReadSize,envcfg:je.log.fileCacheSize,envcfg:je.log.fileMax,envcfg:je.log.fileWarmUpReadSize,envcfg:je.log.fileWarmUpSize,envcfg:je.log.flushNoSyncInterval,envcfg:je.log.flushSyncInterval,envcfg:je.log.fsyncTimeLimit,envcfg:je.log.fsyncTimeout,envcfg:je.log.groupCommitInterval,envcfg:je.log.groupCommitThreshold,envcfg:je.log.iteratorMaxSize,envcfg:je.log.iteratorReadSize,envcfg:je.log.memOnly,envcfg:je.log.nDataDirectories,envcfg:je.log.numBuffers,envcfg:je.log.totalBufferBytes,envcfg:je.log.useNIO,envcfg:je.log.useODSYNC,envcfg:je.log.useWriteQueue,envcfg:je.log.verifyChecksums,envcfg:je.log.writeQueueSize,envcfg:je.maxMemory,envcfg:je.maxMemoryPercent,envcfg:je.maxOffHeapMemory,envcfg:je.nodeDupTreeMaxEntries,envcfg:je.nodeMaxEntries,envcfg:je.offHeap.checksum,envcfg:je.offHeap.coreThreads,envcfg:je.offHeap.evictBytes,envcfg:je.offHeap.keepAlive,envcfg:je.offHeap.maxThreads,envcfg:je.rep.logFlushTaskInterval,envcfg:je.rep.runLogFlushTask,envcfg:je.sharedCache,envcfg:je.stats.collect,envcfg:je.stats.collect.interval,envcfg:je.stats.file.directory,envcfg:je.stats.file.row.count,envcfg:je.stats.max.files,envcfg:je.tree.binDelta,envcfg:je.tree.binDeltaBlindOps,envcfg:je.tree.binDeltaBlindPuts,envcfg:je.tree.compactMaxKeyLength,envcfg:je.tree.maxDelta,envcfg:je.tree.maxEmbeddedLN,envcfg:je.tree.minMemory,envcfg:je.txn.deadlockStackTrace,envcfg:je.txn.dumpLocks,envcfg:je.txn.durability,envcfg:je.txn.serializableIsolation,envcfg:je.txn.timeout,java:args,java:maxMemory,java:minMemory,java:vendor,java:version,je:version,mc:arch,mc:processors,os:name,os:version
```

Comment:
This file should not be added - in fact this directory should not have been added or changed. These seem to be state from a test runs.  Mybe need to update .gitignore for these types of files?

Reply:
Removed